test/UnsafeTest/src_90/org/openj9/test/unsafe/UnsafeTestBase.java

```diff
@@ -30,6 +30,9 @@
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.log4testng.Logger;
+
+import jdk.internal.org.objectweb.asm.*;
+import static jdk.internal.org.objectweb.asm.Opcodes.*;
```

Comment:
Is that possible to import specific classes and field which will be used, rather than import them all?

Reply:
updated.